lib/datadog/core/crashtracking/tag_builder.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require_relative '../utils'
+require_relative '../environment/socket'
+require_relative '../environment/identity'
+require_relative '../environment/git'
+
+module Datadog
+  module Core
+    module Crashtracking
+      # Builds a hash of default plus user tags to be included in a profile
+      module TagBuilder
+        module_function
+
+        def call(settings)
+          # When changing or adding these, make sure they are kept in sync with
+          # https://docs.google.com/spreadsheets/d/1LOGMf4c4Avbtn36uZ2SWvhIGKRPLM1BoWkUP4JYj7hA/ (Datadog internal link)
+          tags = {
+            'host' => Environment::Socket.hostname,
+            'language' => Environment::Identity.lang,
+            'process_id' => Process.pid.to_s,
+            'profiler_version' => Environment::Identity.gem_datadog_version,
```

Comment:
Minor: still mentions profiler

Reply:
Will change, still looking for a good name